### PR TITLE
Add /v1/stats http endpoint for basic metrics

### PR DESCRIPTION
**Pull request by winglot**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -52,3 +65,47 @@ func NewConfigFromFile(path string) (*Config, error) {
 
 	return &config, nil
 }
+
+func (s *Status) MarshalJSON() ([]byte, error) {
+	status := struct {
+		ReloadFailed  int    `json:"reload_failed"`
```
Comment (SebastianCzoch) [inline config/config.go]: why `snake_case` is used instead of `camelCase` ?

File: config/config.go [hunk 2]
```diff
@@ -52,3 +65,47 @@ func NewConfigFromFile(path string) (*Config, error) {
...
+func (s *Status) UpdateReloadFailed() {
+	s.ReloadFailed += 1
+}
+
+func (s *Status) UpdateRenderFailed() {
+	s.RenderFailed += 1
+}
+
+func (s *Status) UpdateLastEvent() {
+	s.LastEvent = time.Now().UTC()
```
Comment (SebastianCzoch) [inline config/config.go]: How you want to test it ?

File: vendor/vendor.json [hunk 3]
```diff
@@ -26,6 +26,18 @@
 			"revision": "53e6ce116135b80d037921a7fdd5138cf32d7a8a",
 			"revisionTime": "2017-01-11T10:11:55Z"
 		},
+		{
```
Comment (SebastianCzoch) [inline vendor/vendor.json]: consider using go dep https://github.com/golang/dep

File: worker/reloader.go [hunk 4]
```diff
@@ -17,6 +17,7 @@ func NewNginxReloaderWorker(client marathon.Marathon, cfg *config.Config) func()
 		defer cfg.Unlock()
 
 		log.Info("Updating nginx config")
+		cfg.Status.UpdateLastEvent()
```
Comment (SebastianCzoch) [inline worker/reloader.go]: not tested ?

File: vendor/vendor.json (hunk 3 above)
Comment (winglot) [reply vendor/vendor.json]: Maybe in separate PR

**Pull request merged**